model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -721,14 +721,36 @@ message StateKey {
     bytes key = 4;
   }
 
+  message MultimapKeysUserState {
```

Comment:
We should document that Get returns keys, Clear removes all keys, Append is an error.

Reply:
User state isn't stored in a PCollection, it is state associated with the transform itself keyed by the currently processing window and key which is how it differs from side inputs since side inputs.

Reply:
Fixed